### PR TITLE
Slowfast network

### PR DESCRIPTION
**Pull request by bryanyzhu**
This PR supports SlowFast network for video action recognition. Paper link is [here](https://arxiv.org/abs/1812.03982).

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1020-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1020/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1020-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1020/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1020-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1020/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1020-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1020/6/index.html 
Code coverage of this PR:  vs. Master:

**Comment by bryanyzhu** [comment]
@hetong007 @zhreshold @chongruo  Please help review, thank you.

File: scripts/action-recognition/train_recognizer.py [hunk 1]
```diff
@@ -256,7 +264,11 @@ def main():
         lr_decay_epoch = [int(i) for i in opt.lr_decay_epoch.split(',')]
     lr_decay_epoch = [e - opt.warmup_epochs for e in lr_decay_epoch]
 
-    optimizer = 'sgd'
+    if opt.slowfast:
```
Comment (hetong007) [inline scripts/action-recognition/train_recognizer.py]: Just curious, how large is the difference if we use 'nag' or 'sgd' for all action-recognition models?

Comment (hetong007) [reply scripts/action-recognition/train_recognizer.py]: Then we may use `nag` for all models

Comment (bryanyzhu) [reply scripts/action-recognition/train_recognizer.py]: Yes, we could. But right now, I haven't test all the models with `nag`. For example, all the 2D TSN models. I can rerun all the models in the future to see if I can use `nag` for everything.

**Comment by mli** [comment]
Job PR-1020-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1020/7/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**